metalearn/metafeatures/metafeatures.json

```diff
@@ -87,6 +87,26 @@
                 "XPreprocessed"
             ]
         },
+        "CategoricalCardinalities": {
```

Comment:
Nice move adding these as resources!

Reply:
I think you are right

Reply:
@epeters3 Would you be willing to fix this and finish cleaning up this PR?

Reply:
Sure thing @bjschoenfeld. Maybe it would be good first though to merge #192? Then along with these refactors mentioned above I can refactor this PR to fit the new caching schema that 192 introduces? The alternative of merging this then merging 192 would require more intense refactoring to be done to 192 to incorporate the changes from this PR I think. It seems like it would be less cumulative work to merge 192 first, then fix merge conflicts in this PR.

Reply:
That sounds great

Reply:
Bless you. I think that one is just waiting a final look over from you. We've moved some of Mason's concerns into separate issues, and I've addressed most of your's and Mason's concerns in subsequent small commits to that branch. There are just one or two unresolved discussions I'd love your quick feedback on there if you have time.